### PR TITLE
Exercise 3: "Promisifying" the ISS locator

### PR DESCRIPTION
**Pull request by Cathe313**

### CODE REVIEW

File: exercise-3.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        }    
+    ).spread(function (response, body) {
+        var userPoz = JSON.parse(body);
+        userLat = Math.round(userPoz.results[0].geometry.location.lat * 100)/100;
+        userLong = Math.round(userPoz.results[0].geometry.location.lng * 100)/100;
+        console.log("You are currently at latitude " + userLat + " x longitude " + userLong + "." ); 
+    }).then(
+        function() {
+            //This code gets the coordinates of the ISS:
+            return request('http://api.open-notify.org/iss-now.json');
```
Comment (ziad-saab) [inline exercise-3.js]: you can simply return this from the previous handler. can you fix please?

**Comment by ziad-saab** [comment]
@Cathe313 This is an OK solution, but I think you can do better! Can you think of a way, using `Promise.join`, to fetch the position of the ISS "at the same time" as prompting the user? These two things are independent and can be joined up later on for the final response :)

**Comment by Cathe313** [comment]
The new version using `Promise.join` is working now!

**Comment by ziad-saab** [comment]
:100:

**Pull request closed without merging**